evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -691,7 +691,7 @@ contract FluxAggregator is AggregatorInterface, Owned {
       rounds[latestRoundId].answer,
       finishedOrTimedOut ? 0 : rounds[_reportableRoundId].startedAt + rounds[_reportableRoundId].details.timeout,
       availableFunds,
-      finishedOrTimedOut ? paymentAmount : rounds[_reportableRoundId].details.paymentAmount
+      rounds[_reportableRoundId].details.paymentAmount
```

Comment:
I believe this is a regression.

Reply:
I probably just need to rebase. I wouldn't have touched this function.